index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Resume Builder</title>
+	</head>
+	<body>
+		<form action="#">
```

Comment:
I used form ID so later I can have javascript do work on the form

I read there might be a way to transform form to JSON // it was a jquery function